File: C_Assignment/Assignment_40/programAQ5.c
///////////////////////////////////////////////////////////
//
//  File name : Assignement40Question5.c
//  Description : Write a recursive program which display below pattern.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

Output : a b c d e f
Prototype :

*/

#include<stdio.h>

void Display()
{
    static char ch = 'a';

    if(ch <= 'f')
    {
        printf("%c\t",ch);
        ch++;
        Display();
    }
}

int main()
{
   Display();

    return 0;
}

File: Java_Assignment/Assignment_59/programAQ5.java
///////////////////////////////////////////////////////////
//
//  File name : Assignement59Question4.java
//  Description : Write java program to print below pattern.
//  Author : Kartik Ganesh Jare
//  Date : 13/11/25
//  
///////////////////////////////////////////////////////////

/*

**********
****  ****
***   ***
**    **
*     *

*/

public class programAQ5
{
    public static void main(String args[])
    {
       int no = 5;

        int i = 0, j = 0;

        for(i = no;i >= 1;i--)
        {
            for(j = 1;j <= i;j++)
            {
                System.out.print("*");
            }
    
            for(j = 1; j <= no - i; j++)
            {
                System.out.print(" ");
            }

            for(j = 1; j <= i;j++)
            {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}